public/pages/CreateMonitor/components/MonitorTimeField/utils/validation.js

```diff
@@ -3,7 +3,11 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+// dateFields is a string array of available time field names
 export const validateTimeField = (dateFields) => (value) => {
+  // Debug use
+  console.log('dateFields: ' + JSON.stringify(dateFields));
+  console.log('value: ' + JSON.stringify(value));
   if (!value) return 'Select a time field';
   if (!dateFields.filter((opt) => opt === value).length)
     return `The field "${value}" does not exist in the selected index mappings, please choose a new time field`;
```

Comment:
Small issue, but I believe the UX convention is to end error messages with periods; e.g., `.`.